docs/python/data-science-tutorial.md

```diff
@@ -0,0 +1,250 @@
+
+- [Miniconda with Python 3.7](https://docs.conda.io/en/latest/miniconda.html)
+
+   > **Note**: If you already have the full Anaconda distribution installed, you don't need to install Miniconda. Alternatively, if you'd prefer not to use Anaconda or Miniconda, you can create a Python virtual environment and install the packages needed for the tutorial using pip. If you go this route, you will need to install the following packages: pandas, jupyter, seaborn, scikit-learn, keras, and tensorflow.
+
+## Set up a data science environment
+
+Visual Studio Code and the Python extension provide a great editor for data science scenarios. With native support for Jupyter notebooks combined with Anaconda, it’s very easy to get started. In this section, you will create a workspace for the tutorial, create an Anaconda environment with the data science modules needed for the tutorial, and create a Jupyter notebook that you’ll use for creating a machine learning model.
+
+1. Begin by creating an Anaconda environment for the data science tutorial. Open an Anaconda command prompt and run `conda create -n myenv python=3.7 pandas jupyter seaborn scikit-learn keras tensorflow` to create an environment named **myenv**. For additional information about creating and managing Anaconda environments, see the [Anaconda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html).
+1. Next, create a folder in a convenient location to serve as your VS Code workspace for the tutorial, name it **hello_ds**.
+1. Open the project folder in VS Code by running VS Code and using the **File** > **Open Folder** command.
+1. Once VS Code launches, open the Command Palette (**View** > **Command Palette** or (`kb(workbench.action.showCommands)`)). Then select the **Python: Select Interpreter** command:
+
+    ![Data Science tutorial: opening the Command Palette in VS Code](images/shared/command-palette.png)
+
+1. The **Python: Select Interpreter** command presents the list of available interpreters that VS Code was able to locate automatically (your list will vary from the one shown below; if you don’t see the desired interpreter see [Configuring Python environments](/docs/python/environments.md)). From the list, select the Anaconda environment you created, which should include the text **‘myenv’: conda**.
+
+   ![Selecting a python environment](images/data-science-tutorial/anaconda-environment.png)
+
+1. Now that an interpreter is selected, run **Terminal: Create New Integrated Terminal** from the Command Palette. This command creates a terminal and automatically activates the selected environment by running its activation script. You’ll notice that the selected environment shows on the left side of the VS Code status bar, and that the “(myenv)” indicator in the terminal indicates you’re using a virtual environment.
```

Comment:
I dont think we need this step, its just an extra step that is not required

Reply:
Ok removed.